test/models/work_test.rb

```diff
@@ -0,0 +1,220 @@
+        work = works(:work1)
+
+        result = work.votes
+
+        expect(result.first).must_be_kind_of Vote
+        expect(result.count).must_equal 2
+      end
+
+      it "has a relationship with users through votes" do
+        work = works(:work2)
+
+        result = work.users
+
+        expect(result.first).must_be_kind_of User
+        expect(result.count).must_equal 2
+      end
+    end
+
+    it "won't be valid without title" do
+      work = works(:work1)
+      work.title = nil
+
+      result = work.valid?
+
+      expect(result).wont_equal true
+    end
+
+    it "will be valid with a title" do
+      result = work.valid?
+      expect(result).must_equal true
+    end
+
+    it "won't be valid without unique title" do
+      new_work = Work.new(title: Work.first.title)
+
+      result = new_work.valid?
+
+      expect(result).wont_equal true
+    end
+
+    it "will be valid with unique title" do
+      new_work = Work.new(title: "workworkworkworkwork", category: "movie", creator: "memyselfandI", publication_year: 1999, description: "test-description")
+
+      new_result = new_work.valid?
+
+      expect(new_result).must_equal true
+    end
+
+    it "will only be valid with correct category" do
+      work = works(:work1)
+
+      result = work.valid?
+      expect(result).must_equal true
+    end
+
+    it "will not be valid without correct category" do
+      work = works(:work1)
+      work.category = "film-reel"
+
+      result = work.valid?
+      expect(result).wont_equal true
+    end
+
+    it "will not be valid without a publication year" do
+      work = works(:work1)
+      work.publication_year = nil
+
+      result = work.valid?
+
+      expect(result).wont_equal true
+    end
+
+    it "will not be valid if year is less than 1700 " do
+      work = works(:work1)
+
+      work.publication_year = 1699
+
+      result = work.valid?
+
+      expect(result).wont_equal true
+    end
+
+    it "will be valid if year is between 1700 and current year" do
+      work = works(:work1)
+
+      work.publication_year = 2010
+
+      result = work.valid?
+
+      expect(result).must_equal true
+    end
+
+    it "will not be valid if year is greater than current year" do
+      work = works(:work1)
+
+      work.publication_year = 2019
+
+      result = work.valid?
+
+      expect(result).wont_equal true
+    end
+
+    it "will not be valid without creator present" do
+      work = works(:work1)
+      work.creator = nil
+
+      result = work.valid?
+
+      expect(result).wont_equal true
+    end
+
+    it "will not be valid without description present" do
+      work = works(:work1)
+      work.description = nil
+
+      result = work.valid?
+
+      expect(result).wont_equal true
+    end
+  end
+
+  describe "self.show_by_vote(category)" do
+    it "returns sorted movies in descending order based on votes" do
+      @books = Work.show_by_vote("book")
+      book_result = @books.length.times do |i|
```

Comment:
Since you don't use the variable `book_result`, you're probably okay to just make the above line `@books.length.times do |i|` without the variable assignment so that you don't have any unused variables